node-playwright-webkit/Dockerfile

```diff
@@ -0,0 +1,364 @@
+FROM debian:bookworm AS build
+
+ARG NODE_VERSION=22.8.0
+
+RUN set -xe; \
+    apt-get -yqq update; \
+    apt-get -yqq install \
+        libatk1.0 \
+        libatk-bridge2.0 \
+        libatomic1 \
+        libcairo-gobject2 \
+        libenchant-2.2 \
+        libepoxy0 \
+        libevent-2.1 \
+        libflite1 \
+        libgdk-pixbuf-2.0 \
+        libgles2 \
+        libgstreamer1.0-0 \
+        libgstreamer1.0-dev \
+        libgstreamer-gl1.0-0 \
+        libgstreamer-plugins-bad1.0-0 \
+        libgstreamer-plugins-base1.0-0 \
+        libgtk-3-0 \
+        libharfbuzz-icu0 \
+        libhyphen0 \
+        liblcms2-2 \
+        libmanette-0.2-0 \
+        libopus0 \
+        libpango-1.0 \
+        libpangocairo-1.0 \
+        libsecret-1-0 \
+        libtinfo6 \
+        libvpx7 \
+        libwebpdemux2 \
+        libwebpmux3 \
+        libwoff1 \
+        libx264-164 \
+        libx264-dev \
+        libxslt1.1 \
+    ; \
+    apt-get -yqq install \
+        curl \
+    ;
+
+RUN set -xe; \
+    curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/master/install.sh | bash; \
+    . ~/.bashrc; \
+    nvm install ${NODE_VERSION} \
+    ;
```

Comment:
Instead of doing this, you can just use the docker image for Node, it's maybe easier

<https://hub.docker.com/_/node>

Reply:
Why would it be easier? What would we gain from that?

Reply:
Less things to install, less libraries to copy, you can just copy the directories and the rest of the libraries

But the work is already done so not that big of a sense to change, if it's an effort

Reply:
For Node, 2 commands in the `Dockerfile` after `FROM scratch` need to be installed: the Node binary and the depending libraries. That's all there is.